### DISCUSSION TITLE
Why is the latest code not in the repository?

### DISCUSSION THREAD

**Discussion by yhete** [openaudible/openaudible#614] [General]
Hi There

Is the version on the repo not the latest as per the description? It says version 3.0 here, but also says so on the website. Surely 3.0 and 3.0 are the same?

The build instruction page is dead as well unfortunately. 

Edit:
**Oh, I see that it's becuase the latest version is not open source, or free. Only the old version is. What a bait and switch!**

**Comment by Ashesh3** [comment]
I had the same realization and decided to come up with https://github.com/openaudible/openaudible/issues/649

**Comment by openaudible** [comment]
We switched from open source to shareware. Asking for donations didn't work. We made a pivot. Price is less than $20 which we think is fair. 
Appreciate it if you @Ashesh3 wouldn't bypass our license model.
